test/integration/gpu/itest_slct_manager_with_gpu.py

```diff
@@ -0,0 +1,130 @@
+import textwrap
+from collections.abc import (
+    Iterator,
+)
+from pathlib import Path
+from tempfile import TemporaryDirectory
+from test.integration.ordinary.test_itde_manager import remove_itde
+
+import pytest
+
+from exasol.nb_connector.ai_lab_config import (
+    Accelerator,
+    AILabConfig,
+)
+from exasol.nb_connector.itde_manager import bring_itde_up
+from exasol.nb_connector.language_container_activation import (
+    open_pyexasol_connection_with_lang_definitions,
+)
+from exasol.nb_connector.secret_store import Secrets
+from exasol.nb_connector.slc import ScriptLanguageContainer
+from exasol.nb_connector.slc.script_language_container import CondaPackageDefinition
+
+
+@pytest.fixture(scope="session")
+def flavor(request) -> str:
+    val = "template-Exasol-8-python-3.10-cuda-conda"
+    return val
+
+
+@pytest.fixture(scope="module")
+def working_path() -> Iterator[Path]:
+    with TemporaryDirectory() as d:
+        yield Path(d)
+
+
+@pytest.fixture(scope="module")
+def secrets_file(working_path: Path) -> Path:
+    return working_path / "sample_database.db"
+
+
+@pytest.fixture(scope="module")
+def sample_slc(
+    slc_secrets: Secrets, working_path: Path, flavor: str
+) -> ScriptLanguageContainer:
+    return ScriptLanguageContainer.create(slc_secrets, name=name, flavor=flavor)
+
+
+@pytest.fixture(scope="module")
+def slc_secrets(secrets_file, working_path, flavor) -> Secrets:
+    secrets = Secrets(secrets_file, master_password="abc")
+    secrets.save(AILabConfig.accelerator, Accelerator.nvidia.value)
+    return secrets
```

Comment:
See fixture `secrets` in [test/conftest.py](https://github.com/exasol/notebook-connector/blob/main/test/conftest.py#L14). 

A problem could be the different scope, as conftest uses default script (I think "function").

Also, the fixture does not use the arguments `working_path` or `flavor`.

I propose: 

```suggestion
@pytest.fixture(scope="module")
def gpu_secrets(secrets) -> Secrets:
    secrets.save(AILabConfig.accelerator, Accelerator.nvidia.value)
    return secrets
```

Reply:
Hm, but the fixture in `conftest.py` has the name "secrets" wheras this fixture here is called "slc_secrets. I actually don't think that the split up is necessary, it would become unnecessary complex.